pkg/version/version.go

```diff
@@ -37,18 +36,39 @@ var (
 	Version string //nolint:gochecknoglobals
 	// FullCommit is a git commit hash.
 	FullCommit string //nolint:gochecknoglobals
-	// CatalogImage is a image path for OLM catalog.
-	catalogImage string //nolint:gochecknoglobals
 )
 
-// CatalogImage returns a catalog image needed for the build of everestctl.
-func CatalogImage() string {
-	catalogImage = devCatalogImage
-	v, err := goversion.NewSemver(Version)
-	if Version != "" && err == nil && v.Prerelease() == "" {
-		catalogImage = fmt.Sprintf(releaseCatalogImage, Version)
+// CatalogImage returns a catalog image name.
+func CatalogImage(v *goversion.Version) string {
+	if isDevVersion() {
+		return devCatalogImage
 	}
-	return catalogImage
+	return fmt.Sprintf(releaseCatalogImage, v)
+}
+
+// ManifestURL returns a manifest URL to install Everest.
+func ManifestURL(v *goversion.Version) string {
+	if isDevVersion() {
+		return devManifestURL
+	}
+	return fmt.Sprintf(releaseManifestURL, v)
+}
+
+func isDevVersion() bool {
+	if Version == "" {
+		return true
+	}
+
+	v, err := goversion.NewVersion(Version)
+	if err != nil {
+		panic(err)
+	}
+
+	if v.Prerelease() != "" && !strings.HasSuffix(v.Prerelease(), "-upgrade-test") {
```

Comment:
a comment to not forget to delete "-upgrade-test"

Reply:
This is here intentionally to support dev releases and at the same time support upgrades from dev releases suffixed with `-upgrade-test`